libraries/AP_HAL_ESP32/Scheduler.cpp

```diff
@@ -243,15 +229,15 @@ void IRAM_ATTR Scheduler::delay_microseconds(uint16_t us)
 void IRAM_ATTR Scheduler::register_timer_process(AP_HAL::MemberProc proc)
 {
 #ifdef SCHEDDEBUG
-    printf("%s:%d \n", __PRETTY_FUNCTION__, __LINE__);
+    DBG_PRINTF("%s:%d \n", __PRETTY_FUNCTION__, __LINE__);
 #endif
     for (uint8_t i = 0; i < _num_timer_procs; i++) {
         if (_timer_proc[i] == proc) {
             return;
         }
     }
     if (_num_timer_procs >= ESP32_SCHEDULER_MAX_TIMER_PROCS) {
-        printf("Out of timer processes\n");
+        DBG_PRINTF("Out of timer processes\n");
```

Comment:
if it is error, then it will be better to rewrite it as GCS_SEND_TEXT(MAV_SEVERITY_ERROR, "Out of timer processes, limit was: %d\n", ESP32_SCHEDULER_MAX_TIMER_PROCS);

Otherwise nobody will see it.

Reply:
You’re not wrong. Like I said in the main comment I think the printf changes should be moved to a separate PR for more focused discussion

Reply:
ok, i reverted DBG_PRINT macro, I'll include it in the next PR.